### DISCUSSION TITLE
Project submission button not generating

### DISCUSSION THREAD

**Discussion by mafaraaz** [github/india#892] [General]
Respected mentors, I have completed all my self-paced learning industrial sessions and project and updated my azure skills on LinkedIn too a long back ago but the student developer pack is not updating and keeps on rejecting me. I reclaimed it multiple times but it's not reclaiming.
I have raised the concern through mail to the frt support multiple times.
I have worked so hard and spent a lot of time doing an internship please update my profile as soon as possible because the last date of project submission is today itself. please it's urgent so that I could submit my project and my hard work.
Anticipating best resolution and happy learning with frt.
 @ey-mentor1 PLEASE ITS URGENT. @ey-mentor1 @ey-mentor1 
My LinkedIn URL: https://www.linkedin.com/in/mohammed-abdul-samad-761476202
![Screenshot (381)](https://user-images.githubusercontent.com/98542941/189996490-45ebe83d-d98c-4c21-98ed-aed3a518b426.png)
![Screenshot (382)](https://user-images.githubusercontent.com/98542941/189996511-6c8ef19b-d4e1-474c-8ce8-99fafc1fbe00.png)
![Screenshot (386)](https://user-images.githubusercontent.com/98542941/189997815-30cdfc91-7857-422c-b5f2-592d7060bd10.png)
![Screenshot (385)](https://user-images.githubusercontent.com/98542941/189997836-59719be9-f475-4438-b813-794bafa138c1.png)
![Screenshot (383)](https://user-images.githubusercontent.com/98542941/189997858-83dfb0ad-b952-4b4c-96d1-1b8ed5c07c7a.png)

**Comment by ghost** [comment]
https://www.linkedin.com/in/sameer-shaik-22a11223a
hello brother , please contact me , I have some doubts on student developer pack and project submission, this is my linked in profile link!
